deps/llvm.mk

```diff
@@ -336,6 +336,7 @@ endif
 ifneq ($(LLVM_LLDB_TAR),)
 	$(JLCHECKSUM) $(LLVM_LLDB_TAR)
 endif
+	-rm -rf $(LLVM_SRC_DIR)
```

Comment:
mind saving everyone time and announce on the mailing list and relevant issues that people should do that if they've already rebuilt in the last few days and have a good build?